backend/.env

```diff
@@ -0,0 +1 @@
+ATLAS_URI=mongodb+srv://jakelop:soen490@cluster0.nsqs3nv.mongodb.net/?retryWrites=true&w=majority
```

Comment:
This repo is public ,we should not post sensitive env files. We should share it on discord instead...

Reply:
resolving as this is outdated